.github/workflows/rspec.yml

```
@@ -29,12 +29,12 @@ jobs:
           key: v2-${{ runner.os }}-gems-fdsh_gateway-${{ hashFiles('**/Gemfile.lock') }}-${{ hashFiles('**/Gemfile' ) }}
           restore-keys: |
             v2-${{ runner.os }}-gems-fdsh_gateway-${{ hashFiles('**/Gemfile.lock') }}-${{ hashFiles('**/Gemfile' ) }}
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node4
```

actionlint (1.7.4)

specifying action "actions/setup-node4" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)